src/ruby.rs

```diff
@@ -1210,6 +1387,12 @@ tests! {
         assert.rb_eq(lazy_eval("::Math::DomainError"), unsafe { rb_eMathDomainError });
     }
 
+    #[test]
+    fn test_types(assert: &mut Assertions) {
+        assert.rs_eq(c_bool(unsafe { RB_TYPE_P("foo".to_ruby(), T_STRING) }), true);
+        assert.rs_eq(c_bool(unsafe { RB_TYPE_P("foo".to_ruby(), T_HASH) }), false);
+    }
```

Comment:
We should be adding tests for all the bindings (like we did with `rb_m*`, `rb_c*` and `rb_e*`), otherwise if nothing is exercising the binding. For things that we don't have a way to test (e.g. `T_FILE`), we should either leave them out for now, or at minimum test that they are within `0x00` and `0x1f` plus a FIXME comment.

Reply:
:+1: